CHANGELOG.md

```diff
@@ -5,6 +5,18 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [24.2.1] - Unreleased
+
+### Changed
+
+-   Reduced minimal infrastructure required to get Rivet running:
+    -   Made K8s Dashboard disabled by default
+    -   Made Prometheus and friends (Vector, Loki, Promtail) disabled by default
+    -   Made Clickhouse disabled by default
+    -   Made NSFW Check API disabled by default
+    -   Made NSFW Check API disabled by default
+    -   Made Image Resizing (via Imagor) disabled by default
```

Comment:
With the addition of `release-please`, the changelog won't be edited directly. I'll try to get this PR set up on my fork to make sure that it will merge properly, but it will be something more like editing the commit message to contain the following in the body:

```
feat(Infra): Reduce minimal infrastructure required to get Rivet running
fix(Infra): Disabled K8s Dashboard by default
fix(Infra): Disabled Prometheus and friends (Vector, Loki, Promtail) by default
fix(Infra): Disabled Clickhouse by default
fix(Infra): Disabled NSFW Check API
fix(Infra): Disabled Image Resizing (via Imagor) by default
```

Ping me if this is ready to go and I haven't got this sorted yet 👍

Reply:
@AngelOnFira ping

Reply:
Sorry, this slipped by in my notifs. As far as I can see, sublists aren't part of the spec, and then because of that, `release-please` wouldn't put that in the changelog.

https://www.conventionalcommits.org/en/v1.0.0/

Reply:
I've made it all one item and put it in the body, so this thread should be ok to close. Not sure if there is a cleaner way to do it, `release-please` does seem to only work in one way without much modification.